src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleared = sourceString.split(';');
+
+  return cleared.reduce((accum, style) => {
+    const n = style
+      .split(':')
+
+      .map((element) => element.trim());
+
+    const [key, value] = n;
+
+    accum[key] = value;
```

Comment:
variant without map(additional sorting)
```suggestion
    if (value) {
      accum[key.trim()] = value.trim();
    }
```